test/integration/controllers/review.spec.js

```diff
@@ -0,0 +1,177 @@
+const { serverInit, serverCleanup } = require('~/test/setup')
+const { REVIEW_NOT_FOUND, UNAUTHORIZED } = require('~/consts/errors')
+const { expectError } = require('~/test/helpers')
+const testUserAuthentication = require('~/utils/testUserAuth')
+
+const endpointUrl = '/reviews/'
+
+let testReview, accessToken
+const nonExistingReviewId = '63bed9ef260f18d04ab15da2'
+
+describe('Review controller', () => {
+  let app, server
+
+  beforeAll(async () => {
+    ;({ app, server } = await serverInit())
+  })
+
+  afterAll(async () => {
+    await serverCleanup(server)
+  })
+
+  describe(`POST ${endpointUrl}`, () => {
+    it('should throw UNAUTHORIZED', async () => {
+      const response = await app.post(endpointUrl).send({
```

Comment:
why don't you assign this review to a variable and keep them all together on the top?

Reply:
fixed